guides/source/active_model_basics.md

```diff
@@ -3,31 +3,54 @@
 Active Model Basics
 ===================
 
-This guide should provide you with all you need to get started using model
-classes. Active Model allows for Action Pack helpers to interact with
-plain Ruby objects. Active Model also helps build custom ORMs for use
+This guide will provide you with what you need to get started using Active
+Model. Active Model provides a way for Action Pack and Action View helpers to
+interact with plain Ruby objects. It also helps to build custom ORMs for use
 outside of the Rails framework.
 
 After reading this guide, you will know:
 
-* How an Active Record model behaves.
-* How Callbacks and validations work.
-* How serializers work.
-* How Active Model integrates with the Rails internationalization (i18n) framework.
+* What Active Model is, and how it relates to Active Record.
+* The different modules that are included in Active Model.
+* How to use Active Model in your classes.
 
 --------------------------------------------------------------------------------
 
 What is Active Model?
 ---------------------
 
-Active Model is a library containing various modules used in developing
-classes that need some features present on Active Record.
+To understand Active Model, you need to know a little about [Active
+Record](active_record_basics.html). Active Record is an ORM (Object Relational
+Mapper) that connects objects whose data requires persistent storage to a
+relational database. However, it has functionality that is useful outside of the
+ORM, some of these include validations, callbacks, translations, the ability to
+create custom attributes etc.
+
+Some of this functionality was abstracted from Active Record to form Active
+Model. Active Model is a library containing various modules that can be used on
+plain Ruby objects that require model-like features but are not tied to any
+table in a database.
```

Comment:
This is great. "What's the difference between Active Record and Active Model?" is a common question I hear. I'm wondering should we drive home the difference between Active Record and Active Model further with something like this: 

"While Active Record provides an interface for defining models that correspond to database tables, Active Model provides functionality for building model-like Ruby classes that don't necessarily need to be backed by a database. Active Model can be used independently of Active Record, and it's not limited to database interactions."

Reply:
I like this a lot, I'll add it. Thanks!